### DISCUSSION TITLE
have a SuppressSecrets  (--suppress-secrets) flag form `helm template`

### DISCUSSION THREAD

**Discussion by acabarbaye** [helmfile/helmfile#583] [Ideas]
`helmfile template` populates and renders the helm charts. It is quite useful for debugging any problem with the charts. It would be really useful if there were an option to hide secrets in the same way as what is done for `helmfile diff` and `helmfile apply`

**Comment by xiaomudk** [comment]
This is a good question. We can discuss it in detail. `helmfile diff` depends on `helm diff`. I think it is more appropriate to implement in `helm diff`.

**Reply by yxxhero** [reply]
maybe should be supported at the helm?
